src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,19 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        ColorSupplier firstSupplier = new ColorSupplier();
+        ColorSupplier secondSupplier = new ColorSupplier();
+        ColorSupplier thirdSupplier = new ColorSupplier();
+        Lottery firstBall = new Lottery();
+        Lottery secondBall = new Lottery();
+        Lottery thirdBall = new Lottery();
+        System.out.println("The first number is... " + firstSupplier.getRandomColor() + " "
+                + firstBall.getRandomBall() + "!");
+        System.out.println("The second number is... " + secondSupplier.getRandomColor() + " "
+                + secondBall.getRandomBall() + "!");
+        System.out.println("The third number is... " + thirdSupplier.getRandomColor() + " "
+                + thirdBall.getRandomBall() + "!");
```

Comment:
1. Please use loop for repetative actions
2. ColorSupplier should be used in Lottery to create ball with initialized color, we don't need to use both services separately